js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/VisJsGraph.jsx

```diff
@@ -0,0 +1,656 @@
+/* global arangoHelper, arangoFetch, $ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { GraphView } from './GraphView';
+import { data } from './data';
+import { UrlParametersContext } from "./url-parameters-context";
+import { AddNodeModal } from './AddNodeModal';
+import { AddEdgeModal } from './AddEdgeModal';
+import { EditNodeModal } from './EditNodeModal';
+import { EditEdgeModal } from './EditEdgeModal';
+import { DeleteNodeModal } from './DeleteNodeModal';
+import { DeleteEdgeModal } from './DeleteEdgeModal';
+import { DocumentInfo } from './DocumentInfo';
+import URLPARAMETERS from "./UrlParameters";
+import { omit, pick, uniqBy } from "lodash";
+
+const VisJsGraph = () => {
+  const currentUrl = window.location.href;
+  const [graphName, setGraphName] = useState(currentUrl.substring(currentUrl.lastIndexOf("/") + 1));
+  let localUrlParameters = URLPARAMETERS;
+  const settingsKey = `${graphName}-gv-urlparameters`;
+  if(localStorage.getItem(settingsKey) !== null) {
+    localUrlParameters = JSON.parse(localStorage.getItem(settingsKey));
+  }
```

Comment:
this should be in a useEffect

Reply:
After discussing this comment with @palashkaria we agreed to take care of this later.